### PR TITLE
Remove class from sig util

### PR DESCRIPTION
**Pull request by forbesashli**
### Changelist 
Changed the signal util to not be a class

### Resolved Tickets
There is on issue on jira this resolves

### CODE REVIEW

**Review by gtaharaedmonds — APPROVED** [review]
i trust

File: scripts/canup
Comment (Lucien950) [inline scripts/canup:1]: ?

File: software/tracksight/backend/app/process/flask_apps/socket_app.py [hunk 1]
```diff
@@ -9,9 +9,8 @@
 
 logger = logging.getLogger("telemetry_logger")
 
-from ..signal_util import SignalUtil
+from .. import signal_util as signal_util
```
Comment (Lucien950) [inline software/tracksight/backend/app/process/flask_apps/socket_app.py:12]: uh

File: software/tracksight/backend/app/telemetry.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 from process.flask_apps.http_app import app as http_app
 from process.flask_apps.socket_app import socketio
 from flask_cors import CORS
-from process.signal_util import SignalUtil
+from process import signal_util as signal_util
```
Comment (Lucien950) [inline software/tracksight/backend/app/telemetry.py:20]: hmm

**Review by Lucien950 — APPROVED** [review]
sinon c'est d'accord

File: scripts/canup
Comment (gtaharaedmonds) [reply scripts/canup:1]: i reverted this

**Pull request merged**